apps/all/src/main/content/jcr_root/apps/dx/configs/config/com.adobe.dx.responsive.internal.ResponsiveServiceImpl.config

```diff
@@ -0,0 +1 @@
+breakpoints=["Mobile", "Tablet", "Desktop"]
```

Comment:
❤️
Can we do key value pairs here so we can kill the other pixel value requirement?

mobile: 599
tablet: 1199
desktop: {infinity}

Reply:
let's change this in another PR as this is just me doing too much in that one